### PR TITLE
Dots for island countries

### PR DESCRIPTION
**Pull request by Flizzyclone**
This PR changes the tiny island countries on the world maps into larger dots so you can actually see/find them, like Mapshaper does. It also renames the map to "World Countries (UN)" from world countries, so we can differentiate when we have another map that includes Taiwan, etc.

### CODE REVIEW

**Review by Fish1 — COMMENTED** [review]
Isn't this map from the World Bank? Not the UN?

**Comment by Flizzyclone** [comment]
> Isn't this map from the World Bank? Not the UN?

It's kind of weird. The shapefile is from the World Bank but it is depicting UN-recognized countries. The World Bank shapefile on it's own is not very specific and has tons of autonomous areas and unclaimed areas considered to be countries.

File: apps/yapms/src/lib/assets/maps/glb/glb-countries-2020078-blank.svg [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
...-.05-.1-.14-.06-.12-.12-.12-.34-.14-.15-.2-.07-.48-.04zm312.3 1.52.06.38.16.29.12.12.22.06.22-.08.04-.09.02-.5-.17.07h-.19l-.1-.19-.38-.06zm-1.07 3.5-.07.06-.02.25.25.08.04-.3-.2-.09zm-866.08 15.4-.02.17.12.05.05-.16-.15-.06zm616.34 33.94-.11.02-.05.1.14.17.14-.1-.12-.19zm-75.76 30.78-.26.08.03.16.2.1.1-.21-.07-.13zm4.07 1.14-.19.14.11.3.33.05.12-.16-.1-.13-.27-.2zm47.84 9.63-.1.22.03.2-.1.22-.29.1.24.42-.18.1v.2l-.16.13.05.27.22.16-.17.43.37.17-.19.17.13.44-.22.22.05.1-.18.37-.04.35.2.18.36-.02.28-.1.14-.37.17-.08.14-.4.15.3.26.02.26.28.34.06.2.09.14-.12v-.24l-.11-.3.18-.05.11.09.03.36.19.32.2.1.23-.02.31.07.08-.1.26-.02-.05-.24.27-.24-.08-.2-.44-.09-.25.06.05.23-.48-.06-.22-.12v-.34l.07-.24.17.2.13-.05-.14-.38.33.07.34-.09.3.1.12.33.24-.04.05-.1-.23-.12.13-.17.3.08.25-.38-.02-.36-.07-.37-.32-.17h-.2l-.17.09-.22.34-.3-.15-.36.16-.29.21.02.13-.2.06-.21.23-.6-.3-.14-.17.07-.2.48-.1.04-.27-.25-.06.08-.18-.21-.05-.14.37-.1-.29-.23.29-.39.17-.1-.05-.02-.28-.18-.22-.13.2.03.3-.2-.08.07-.34.17-.16.22-.54-.24-.23-.15.28-.1-.25.36-.3-.15-.24h-.29zm.99.94-.34.4.04.42.3-.07.02-.25.25-.35-.27-.15zm-1.78 1.8.05.17.29-.06-.34-.11z" region="FRA" value="1" short-name="France" long-name="France"/>
```
Comment (Fish1) [inline apps/yapms/src/lib/assets/maps/glb/glb-countries-2020078-blank.svg]: France should be region="20" not FRA.

**Review by Fish1 — CHANGES_REQUESTED** [review]

File: apps/yapms/src/lib/assets/maps/glb/glb-countries-2020078-blank.svg (hunk 1 above)
Comment (Flizzyclone) [reply apps/yapms/src/lib/assets/maps/glb/glb-countries-2020078-blank.svg]: great catch

**Review by Fish1 — APPROVED** [review]
LGTM

**Pull request merged**